indicators/dependency_management.json

```diff
@@ -0,0 +1,19 @@
+{
+  "@context": "https://w3id.org/everse/rsqi#",
+  "@id": "https://w3id.org/everse/i/indicators/dependency_management",
+  "@type": "SoftwareQualityIndicator",
+  "name": "Software has dependency management solution",
+  "identifier": "https://w3id.org/everse/i/indicators/dependency_management",
+  "description": "Reviews how external libraries and dependencies are managed to ensure compatibility and security.",
```

Comment:
@fdiblen since there is no source, I guess you are defining this indicator yourself, right?
What is it exactly that you are measuring? what does it mean to "pass" this indicator? I don't understand very well how the review happens. Sorry to be difficult, but I think we should be clear on our own definitions.